File: Beecrowd/Python/bee1117.py
scoreSum = 0
count = 0

while True:
    score = float(input())
    
    if  0 <= score <= 10:
        scoreSum += score
        count+=1
    else:
        print("nota invalida")
    
    if count == 2:
        print(f"media = {(scoreSum / 2):.2f}")
        break

File: Beecrowd/C/bee1117.c
#include <stdio.h>

int main() {
  float score, sumOfScores = 0;
  int count = 0;

  while (count < 2) {
    scanf("%f", &score);

    if (score >= 0 && score <= 10) {
      sumOfScores += score;
      count += 1;
    } else
      printf("nota invalida\n");
  }

  printf("media = %.2f\n", sumOfScores / 2);

  return 0;
}
